nilearn/input_data/nifti_masker.py

```diff
@@ -199,8 +199,8 @@ def fit(self, imgs=None, y=None):
             self.affine_ = self.mask_img_.get_affine()
         # Load data in memory
         self.mask_img_.get_data()
-        if self.verbose > 10:
-            print "[%s.fit] Finished fit" % self.__class__.__name__
+        if self.verbose > 0:
```

Comment:
Sorry, I thought it was previously requested that I change this.  All of the similar prints from within this function were for `verbose > 0`, so I thought I was asked to tweak this one while I was there.

Please confirm either way, thanks!